book/src/pong/09_keeping_score.md

```diff
@@ -28,23 +28,35 @@ If you lose while you have 0 lives, you lose the game.
 Firstly, let's add the score to the `Paddle` objects:
 
 ```rust
-struct Paddle {
+pub struct Paddle {
     pos: Vector2D<Fixed>,
     health: i32,
 }
 ```
 
 and in the `new()` function, initialise it to 3.
 
+We are gonna use it for displaying health as hearts and decrementing it when ball touches either left side or right side of screen. Add these functions to `Paddle` struct.
```

Comment:
```suggestion
We're going to use it for displaying the health as hearts and decrementing it when ball touches the side of the screen. Add these functions to `Paddle` struct.
```

@gwilymk might have a better suggestion.

Reply:
Thanks! That sounds much better.